app/backend/src/couchers/migrations/versions/f67a7883d01a_reconciling_migrations.py

```diff
@@ -0,0 +1,83 @@
+"""Reconciling migrations
+
+Revision ID: f67a7883d01a
+Revises: 27a2782784d0
+Create Date: 2021-04-02 17:33:57.574539
+
+"""
+import geoalchemy2
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "f67a7883d01a"
+down_revision = "27a2782784d0"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "language_abilities",
+        sa.Column("id", sa.BigInteger(), nullable=False),
+        sa.Column("user_id", sa.BigInteger(), nullable=False),
+        sa.Column("language_code", sa.String(length=3), nullable=False),
+        sa.Column(
+            "fluency",
+            sa.Enum("say_hello", "beginner", "intermediate", "advanced", "fluent", "native", name="languagefluency"),
+            nullable=False,
+        ),
+        sa.ForeignKeyConstraint(["user_id"], ["users.id"], name=op.f("fk_language_abilities_user_id_users")),
+        sa.PrimaryKeyConstraint("id", name=op.f("pk_language_abilities")),
+        sa.UniqueConstraint("user_id", "language_code", name=op.f("uq_language_abilities_user_id")),
+    )
+    op.create_index(op.f("ix_language_abilities_user_id"), "language_abilities", ["user_id"], unique=False)
+    op.create_table(
+        "regions_lived",
+        sa.Column("id", sa.BigInteger(), nullable=False),
+        sa.Column("user_id", sa.BigInteger(), nullable=False),
+        sa.Column("region_code", sa.String(length=3), nullable=False),
+        sa.ForeignKeyConstraint(["user_id"], ["users.id"], name=op.f("fk_regions_lived_user_id_users")),
+        sa.PrimaryKeyConstraint("id", name=op.f("pk_regions_lived")),
+        sa.UniqueConstraint("user_id", "region_code", name=op.f("uq_regions_lived_user_id")),
+    )
+    op.create_index(op.f("ix_regions_lived_user_id"), "regions_lived", ["user_id"], unique=False)
+    op.create_table(
+        "regions_visited",
+        sa.Column("id", sa.BigInteger(), nullable=False),
+        sa.Column("user_id", sa.BigInteger(), nullable=False),
+        sa.Column("region_code", sa.String(length=3), nullable=False),
+        sa.ForeignKeyConstraint(["user_id"], ["users.id"], name=op.f("fk_regions_visited_user_id_users")),
+        sa.PrimaryKeyConstraint("id", name=op.f("pk_regions_visited")),
+        sa.UniqueConstraint("user_id", "region_code", name=op.f("uq_regions_visited_user_id")),
+    )
+    op.create_index(op.f("ix_regions_visited_user_id"), "regions_visited", ["user_id"], unique=False)
+    op.drop_index("idx_events_geom", table_name="events")
+    op.drop_index("idx_nodes_geom", table_name="nodes")
+    op.drop_index("idx_page_versions_geom", table_name="page_versions")
+    op.add_column("users", sa.Column("avatar_filename", sa.String(), nullable=True))
+    op.drop_index("idx_users_geom", table_name="users")
```

Comment:
These should all be removed. The `_geom` indexes shouldn't be dropped, and `avatar_filename` shouldn't have been added, seems to be from a merge issue